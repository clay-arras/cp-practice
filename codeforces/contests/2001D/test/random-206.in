1
3
21 10 13
