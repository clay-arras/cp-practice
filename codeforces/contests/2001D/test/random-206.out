3
21 10 13
